### ISSUE TITLE
fix Apache2 and BSD license issues

### ISSUE THREAD

**Issue by ccoVeille** [jargonLint/jargonLint#42]
See 
- #41
- #39 

Some

              But 

- BSD-licensed
- BSD licensed
- BSD license
- BSD licenses
- BSD licensed
- license BSD
- licenses BSD
- licensed BSD
- dual-licensed BSD

These are license based, so can be added to the file by using look behind or forward regexp.

Funny thing about license/license US/UK

So we have to duplicate with license

BSD the OS, can also be found with slash

Linux/BSD

So maybe we will have to move terms with multiple meaning to a separate file

_Originally posted by @ccoVeille in https://github.com/jargonLint/jargonLint/pull/39#discussion_r1666006064_
         

              This one might cause problem with Apache2 being a web server.
              This one might need combination of license keywords and variations like BSD

_Originally posted by @ccoVeille in https://github.com/jargonLint/jargonLint/pull/39#discussion_r1665979819_

**Comment by alexvonme** [comment]
I don't think this is much of a problem really BSD the OS is not only increasingly uncommon, it's usually expressed in terms that won't coincide with this rule such as ``FreeBSD`` and ``OpenBSD``.

On Apache, the webserver ``Apache 2.0`` [hasn't had a release in a decade](https://en.wikipedia.org/wiki/Apache_HTTP_Server#Development), and as such, I don't think we need to consider it an issue. In the overwhelming majority of cases, ``Apache 2.0``, will refer to the license.

(Note: This could also be a soft rule for the wider repository, that we don't feel the need to take into account decade old and unsupported software)

**Comment by ccoVeille** [comment]
I agree with you about BSD.

For Apache2 webserver, I would say. we shouldn't touch it when we find it written as "apache2".

But there is no rush to add this one I think

**Comment by alexvonme** [comment]
Sure, Apache is not an often miswritten license anyway, unlike the creative commons licenses.

**Comment by alexvonme** [comment]
Can we close this? You added the relevant commit here, https://github.com/jargonLint/jargonLint/pull/41/commits/b3062b25779758fdcaa2316d302ef8a648b3a3a5

**Comment by ccoVeille** [comment]
> Can we close this? You added the relevant commit here, https://github.com/jargonLint/jargonLint/pull/41/commits/b3062b25779758fdcaa2316d302ef8a648b3a3a5

Not yet, the idea is to fix apache2 at some point.

I would like to open a PR, but as I said there is no rush about it

**Comment by ccoVeille** [comment]
But we can merge as is for now.
